golang/.ci/go-cloud-run-hello-world.cloudbuild.yaml

```diff
@@ -3,6 +3,7 @@ steps:
 - id: 'Build Container Image'
   name: 'gcr.io/cloud-builders/docker:latest'
   args: ['build', '-t', 'gcr.io/${PROJECT_ID}/${_SERVICE_TAG}:${COMMIT_SHA}', '.']
+  dir: 'golang/go-cloud-run-hello-world'
```

Comment:
The instructions in golang/CONTRIBUTING.md are for this to be run from the golang directory. Is it possible to run the trigger in that same context?

Reply:
I tried to look at the trigger to see if you can specify the context outside of the yaml file and you can't. Hence this change.